dashboard/config/scripts/levels/Music Level 1.level

```diff
@@ -20,10 +20,10 @@
         },
         {"message": "Please play one sound."}
       ],
-      "starterCode": {
+      "defaultCode": {
```

Comment:
Thanks for raising this! `start_blocks` is indeed what we use for XML. I'm still trying to determine what we call the JSON serialization, but I'm growing convinced that talking about it as "blocks" isn't accurate enough. The JSON serialization also takes care of state related to global variables, plugins, and (when we get there) procedure definitions that are defined on separate workspaces altogether. 

Given that we call `Blockly.serialization.workspaces.load`, I would suggest `start_workspace_json` or something similar.

I think `start_blocks` still works for XML because it is literally just an XML tree of blocks. And while Google Blockly also supports `Blockly.serialization.blocks.save` for serializing individual blocks, I don't think we'd ever want to use that.

Reply:
Yep good call - also given the fact that we use `start_sources` for other levels (e.g. Javalab), I think a more generic name makes sense here as well.

Reply:
Per offline discussion, using `startSources`.